device-side-launch/dsl.c

```diff
@@ -144,7 +144,7 @@ char *generatePTX(const char *ll, size_t size, const char *filename)
     nvvmProgram program;
     size_t PTXSize;
     char *PTX = NULL;
-    const char *options[] = { "-arch=compute_35" };
+    const char *options[] = { "-arch=compute_52" };
```

Comment:
With the updated change above, we're now out-of-sync with the compiler target here.